packages/config/src/Config.ts

```diff
@@ -294,31 +296,31 @@ export class Config {
      * App name used in config filenames (e.g., *my_cli*.config.json)
      */
     public get appName(): string {
-        return this.mApp;
+        return path.isAbsolute(this.mApp) ? ImperativeConfig.instance.loadedConfig?.name ?? path.basename(this.mApp) : this.mApp;
```

Comment:
Not sure if this is a real problem or it's just something that will only happen during integration testing.

When calling the a CLI using a method like `node --require ts-node/register <absolute-path-to-CLI.ts> config init`, some Config APIs may have issues saving the config and schema files.

I'm more than happy to revert this change back if we think that it's something people shouldn't be doing. 😋  🦆